crash_similarity.py

```diff
@@ -96,6 +107,11 @@ def get_stack_trace_for_uuid(uuid):
 
 
 def train_model(corpus):
+    # Store the time of training the model in last_trained.txt
+    cur_time = datetime.datetime.today()
+    with open("last_trained.txt", "w") as text_file:
+        text_file.write(cur_time.strftime('%b %d %Y %I:%M%p'))
```

Comment:
No need for the local variable `cur_time`, you can directly do `datetime.datetime.today().strftime(...)`.